### PR TITLE
Multi directory rename precedence

### PR DESCRIPTION
**Pull request by insidewhy**

### CODE REVIEW

**Review by tkinzer — APPROVED** [review]
Cooolio

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: circleci/node:10
+      - image: circleci/node:14
```
Comment (sxwei123) [inline .circleci/config.yml:5]: Should use `cimg/node:14`. 
https://circleci.com/developer/images/image/cimg/node.

Comment (sxwei123) [reply .circleci/config.yml:5]: Maybe use with multiple versions of node also.

**Review by irot — APPROVED** [review]
Example 5's a doozy. Awesome stuff.

**Review by owcs-solx — APPROVED** [review]

**Pull request merged**